config/initializers/modify_rails_form_builder.rb

```diff
@@ -0,0 +1,14 @@
+class ActionView::Helpers::FormBuilder
+  def error_message_for(field_name)
```

Comment:
This is a little bit scary! I would love to avoid requiring an initializer for this. Did you consider switching to something like `simple_form`?